### DISCUSSION TITLE
CreatedBy auto-column not populated in "App Action" automation

### DISCUSSION THREAD

**Discussion by pandel** [Budibase/budibase#7346] [Help] [answered]
Hi!

I set up an "App Action" automation which is creating a row in a Budibase DB table. This table is setup with auto-columns Created By and Created At.

When the automation is triggered I get a new row in the table, but "Created by" stays empty.

Why? I understand that there might be no user information for a CRON or whatever automation type, but if I tie a "App Action" to i.e. a button it should be easy to identify who is triggering the automation.

I know, I can simply add an automation field and set its value to the current user id, but I am curious if this is expected behaviour or not :-)

Regards,
Holger

**Comment by melohagan** [comment, marked as answer]
Hi @pandel 

This is expected behaviour. Automations have no concept of current user as they are run in the background. 

As you mentioned you can pass through the current user as a binding! However it's worth noting that just because a user requested a row be created, does not mean that user created the row - the automation did. 
Depends on what you're looking for!

**Reply by pandel** [reply]
I understand. I solved it by passing the current user. That works really well.
